Cxx11/stencil-sycl-usm.cc

```diff
@@ -307,7 +301,7 @@ int main(int argc, char * argv[])
   }
 
   try {
-    sycl::queue q(sycl::gpu_selector{});
+    sycl::queue q(sycl::gpu_selector{}, sycl::property::queue::in_order{});
```

Comment:
`sycl::queue q{sycl::gpu_selector{}, sycl::property::queue::in_order{}};`

Reply:
i just remembered i made this change on a different branch and need to redo it on the right one.